src/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -13,211 +13,147 @@
 
 using System.Collections.Generic;
 using System.IO;
+using System.Runtime.ExceptionServices;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace System.Diagnostics
 {
-    internal sealed class AsyncStreamReader : IDisposable
+    internal sealed class AsyncStreamReader
     {
-        internal const int DefaultBufferSize = 1024;  // Byte buffer size
+        private const int DefaultBufferSize = 1024;  // Byte buffer size
```

Comment:
I do not have any historical context on this but in general since the code is shared across different platforms I have seen us being conservative in buffer sizes. Do you know how much of a performance gain are we likely to see? We can surely consider making the changes depending on the gain.